src/coverlet.core/CoverageDetails.cs

```diff
@@ -8,7 +8,18 @@ public class CoverageDetails
         public int Total { get; internal set; }
         public double Percent
         {
-            get => Math.Round(Total == 0 ? 1 : Covered / Total, 3);
+            get => Math.Round(Total == 0 ? 1 : Covered / Total, 4);
+        }
+
+        public double GetCoveragePercentage()
```

Comment:
Please use property `CoveragePercentace { get }`

Reply:
Yeah, I can do that. Will convert method into prop CoveragePercentage with getter only.